docs/tutorials/elasticdl_on_prem_cluster.md

```diff
@@ -12,7 +12,7 @@ export TRAVIS_BUILD_DIR=$PWD
 bash scripts/travis/build_images.sh
```

Comment:
Commands here need to be updated too. Or we plan to update it in another PR?

Reply:
Changed.